src/CMake/config/postinst.in

```diff
@@ -26,6 +26,10 @@
 # 3. When re-installing (say from 2.2.0 to 2.2.0) then prerm is NOT run but
```

Comment:
Copyright date needs to updated (line 2). In this case, a new line needs to be added:
`Copyright (c) 2022 Advanced Micro Devices, Inc.`